programmers/다트-게임.js

```diff
@@ -0,0 +1,40 @@
+function solution(dartResult) {
+    const turn = [];
+    let num = 0;
+    let answer = [];
```

Comment:
배열,객체는 const 선언을 해야한다.
배열의 변수명은 복수형으로 선언 해야한다.
```suggestion
    const answers = [];
```